aws/logs_monitoring/template.yaml

```diff
@@ -52,6 +53,17 @@ Parameters:
     Type: String
     Default: ""
     Description: DO NOT CHANGE unless you know what you are doing. Override the default location of the function source code.
+  InstallFromLayer:
```

Comment:
I left some comments about these parameters on the README file, but those comments apply to the template as well.